api/v1alpha1/application_types.go

```diff
@@ -27,7 +27,7 @@ const ApplicationKind = "Application"
 // ApplicationSpec defines the desired state of Application
 type ApplicationSpec struct {
 	// ConfigRepo is the address of the git repository containing the automation for this application
-	ConfigRepo string `json:"config_repo,omitempty"`
+	ConfigRepo string `json:"config_url,omitempty"`
```

Comment:
I think we should rename the Field itself too, having it in the yaml only will make it even more confusing.

Reply:
I think it would be more confusing having more places with ConfigURL than just this single one

Reply:
do you mind if we merge this now? and discuss it later?. I think we can come up with an even better solution for all cases. Renaming this won't be the final solution for all the cases involved.

Reply:
@luizbafilho ^